Common/Source/SaveLoadTask/SaveTask.cpp

```diff
@@ -43,7 +43,8 @@ void RenameIfVirtual(const unsigned int i) {
   if ( _tcslen(WayPointList[i].Name)> (NAME_SIZE-5)) return;
 
   TCHAR tmp[NAME_SIZE+10];
-  _stprintf(tmp,_T("TSK_%s"),WayPointList[i].Name);
+  if(_tcsncmp( WayPointList[i].Name,_T("TSK_"), 1)!=0)
+    _stprintf(tmp,_T("TSK_%s"),WayPointList[i].Name);
   LK_tcsncpy(WayPointList[i].Name,tmp,NAME_SIZE);
```

Comment:
2 bug here : 
- you compare only first character of  tp name so any turnpoint with name like  "Txxxx" are not renamed.
- if condtion line 46 is false, you have segfault line 48 because "tmp" is uninitialized.